Package.swift

```diff
@@ -31,7 +31,7 @@ let package = Package(
   dependencies: [
     .package(
         url: "https://github.com/google/abseil-cpp-binary.git",
-        "1.2022062300.1" ..< "1.2022062400.0"
+        "1.2024011601.0" ..< "1.2024011700.0"
```

Comment:
See https://github.com/google/abseil-cpp-binary/pull/7

Note that for Abseil, Abseil's original patch version is moved into the minor version, and the patch is freed for us to make infrastructure patches.